build/room_ensurer/room_ensurer.py

```diff
@@ -128,33 +128,36 @@ def ensure_rooms(self) -> None:
                 PATH_FINDING_BROADCASTING_ROOM,
             ]:
                 try:
-                    self._ensure_room_for_network(network, alias_fragment)
+                    room_alias_prefix = make_room_alias(ChainID(network.value), alias_fragment)
+                    self._first_server_actions(room_alias_prefix)
+                    log.info(f"Ensuring {alias_fragment} room for {network.name}")
+                    self._ensure_room_for_network(room_alias_prefix)
```

Comment:
This introduces a change in behaviour. Previously, the `_first_server_actions` part ended with a `return`. Now, the remaining `_ensure_room_for_network` code is executed even when the room is created in `_first_server_actions`. Is this intentional?

Reply:
Yes, it will simply run through the `_ensure_room_for_network` and warn that the room does not exist on other servers. Not executing the code as it was before, was made under the assumption that if the first server creates the room the others should not be joined yet, so there is nothing to check. 
But a malicious other server could also have created a room with the same alias and this should trigger a warning for the first server.